src/background.js

```diff
@@ -29,50 +29,74 @@ function sendVideoUrlToMetubeAndSwitchTab(videoUrl, metubeUrl, tab) {
+    chrome.storage.sync.get(['metube', 'contextMenuClickBehavior'], function(data) {
         if (data === undefined || !data.hasOwnProperty('metube') || data.metube === "") {
             openTab(chrome.runtime.getURL('options.html'), tab);
             return
         }
-        sendVideoUrlToMetubeAndSwitchTab(item.linkUrl, data.metube, tab);
+        if ( data.contextMenuClickBehavior == 'context-menu-send-current-url') {
+            sendVideoUrlToMetube(item.linkUrl, data.metube);
+        } else if (data.contextMenuClickBehavior == 'context-menu-send-current-url-and-switch'){
+            sendVideoUrlToMetube(item.linkUrl, data.metube, function() {
+                openTab(data.metube, tab);
+            });
+        } else {
+            console.log("Unknown contextMenuClickBehavior value: " + data.contextMenuClickBehavior);
+        }
     });
 });
 
 chrome.action.onClicked.addListener(function(tab) {
-    chrome.storage.sync.get(['metube', 'sendOnClick'], function(data) {
+    chrome.storage.sync.get(['metube', 'clickBehavior'], function(data) {
         if (data === undefined || !data.hasOwnProperty('metube') || data.metube === "") {
             openTab(chrome.runtime.getURL('options.html'), tab);
             return
         }
-        chrome.tabs.query({
-            active: true,
-            lastFocusedWindow: true
-        }, function(tabs) {
-            // use this tab to get the youtube video URL
-            let videoUrl = tabs[0].url;
-            if (data.sendOnClick) {
-                sendVideoUrlToMetubeAndSwitchTab(videoUrl, data.metube, tab);
-            }
-            else {
-                console.log("Going to Metube URL...");
-                openTab(data.metube, tab);
-            }
-        });
+        if (data.clickBehavior == 'do-nothing') {
+            return
+        } else if (data.clickBehavior == 'go-to-metube') {
+            console.log("Going to Metube URL...");
+            openTab(data.metube, tab);
+        } else if (data.clickBehavior == 'send-current-url') {
+            chrome.tabs.query({
+                active: true,
+                lastFocusedWindow: true
+            }, function(tabs) {
+                // use this tab to get the youtube video URL
+                let videoUrl = tabs[0].url;
+                sendVideoUrlToMetube(videoUrl, data.metube);
+            });
+        } else if (data.clickBehavior == 'send-current-url-and-switch') {
+            chrome.tabs.query({
+                active: true,
+                lastFocusedWindow: true
+            }, function(tabs) {
+                // use this tab to get the youtube video URL
+                let videoUrl = tabs[0].url;
+                sendVideoUrlToMetube(videoUrl, data.metube, function() {
+                    openTab(data.metube, tab);
+                });
+            });
+        } else {
+            console.log("Unknown clickBehavior value: " + data.clickBehavior);
```

Comment:
for users that condition is bad practice. because if they get something error they will not read `console` for debug and will think "it's just not working".

i think we need use some default behavior and send url to metube with `openTab` if we can't do something another.

Reply:
i suggest use `send-current-url-and-switch` as default behavior

Reply:
The current behavior on extension icon click is just opening the metube URL, so I preserved that instead of choosing another default. This was to prevent surprises for longtime users after the extension upgrade as they may be used to the older behavior if they've been using the extension.

If you still insist, I can sure change the default click behavior - it's described over here:  https://github.com/Rpsl/metube-browser-extension/pull/7/commits/1a12fc0c99a27194a27ae7838536110f5c9a97de#diff-70b26bb50b16514f30f51e300930a4e23d2acfb8bbea7ece91a3ed037c98c07aR38 

I use the word "checked" in the input tag to check a radio button by default for the user there.

>> if they get something error they will not read console for debug and will think "it's just not working".
I agree - so what do you propose the right way to bubble up error would be? just using alert as the old code suggested was throwing ReferenceError, which is why I went for console.log instead.